content/local/store.go

```diff
@@ -459,8 +484,10 @@ func (s *store) Writer(ctx context.Context, opts ...content.WriterOpt) (content.
 	if wOpts.Ref == "" {
 		return nil, fmt.Errorf("ref must not be empty: %w", errdefs.ErrInvalidArgument)
 	}
+
 	var lockErr error
-	for count := uint64(0); count < 10; count++ {
```

Comment:
why not just remove the for-loop here? The opt is not visible to end-user and by default it's not retry, rigtht?

Reply:
Did you mean directly remove the option, and no retry here?
I don't know if anyone will use this directly.... so I left a option here ...

Reply:
Yes. I think we can remove it directly. @dmcgowan WDYT?

Reply:
Remove the loop and option makes more sense. I don't see this as something that would be configurable. Just need to figure out where the correct place to retry is in the higher level logic. What we have may be enough already.